database/migrations/2023_12_14_034503_create_users_v2_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUsersV2Table extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropUnique(['email']);
+
+            $table->increments('id')->change();
+            $table->string('name', 255)->change();
+            $table->text('password')->change();
+            $table->string('email', 320)->unique()->change();
+            $table->integer('role_id')->default(1)->after('email');
+            $table->integer('status_id')->default(1)->after('role_id');
+            $table->boolean('verify')->default(false)->after('status_id');
+            $table->date('email_verified_at')->nullable()->change();
+            $table->string('avatar', 255)->nullable()->after('verify');
+            $table->tinyInteger('gender')->nullable()->after('avatar');
+            $table->date('birthday')->nullable()->after('gender');
+            $table->rememberToken()->nullable()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('users');
+
+        Schema::create('users', function (Blueprint $table) {
```

Comment:
```suggestion
        Schema::table('users', function (Blueprint $table) {
```
xóa các cột tạo mới, khôi phục index bị xóa, ko làm ntn nha.